src/main/java/com/josdem/vetlog/binder/UserBinder.java

```diff
@@ -34,7 +34,7 @@ public User bindUser(Command command) {
         user.setRole(Role.USER);
         user.setFirstName(userCommand.getFirstname());
         user.setLastName(userCommand.getLastname());
-        user.setPhone(userCommand.getPhone());
+        user.setMobile(userCommand.getPhone());
         user.setEmail(userCommand.getEmail());
```

Comment:
We had already a column to store user's phone named `mobile`, so I changed to avoid duplication.